### PR TITLE
Block: "people-intro" #153

### LINKED ISSUE #153: Block: "people-intro"
For https://sgcreative.southernglazers.com/bloominbrands/p/8

Screenshot 2023-07-17 at 4 36 40 PM

### PR DESCRIPTION
**Pull request by atopper**
## Issue

Fix #153

## 🔗 Test URLs:

- Before: https://main--sgws-creative--hlxsites.hlx.page/bloominbrands/p/8
- After: https://people-intro153--sgws-creative--hlxsites.hlx.page/bloominbrands/p/8
- After: https://people-intro153--sgws-creative--hlxsites.hlx.page/bloominbrands/fragments/simi-people

Original: https://sgcreative.southernglazers.com/bloominbrands/p/8

### CODE REVIEW

**Comment by tmathern** [comment]
Scaling is off on small screens...
Screenshot 2023-07-19 at 1 42 16 PM

File: blocks/vertical-selector/vertical-selector.js [hunk 1]
```diff
@@ -139,12 +139,17 @@ export default async function decorate(block) {
...
-            // setupCharting(block);
             const nextViewer = createTag('div', { class: `fragment-viewer-${rowIndex} animate` });
-            nextViewer.append(viewerHeader);
+
+            // Do not carry the title to the top of the viewed content, if specified.
+            if (!block.classList.contains('no-title')) {
+              const viewerHeader = createTag('h2', {});
+              viewerHeader.innerHTML = columns[0].innerHTML;
+
+              // setupCharting(block);
```
Comment (tmathern) [inline blocks/vertical-selector/vertical-selector.js:149]:
```suggestion
```
(Drop the comment)

File: scripts/scripts.js [hunk 2]
```diff
@@ -402,6 +402,52 @@ export function decorateSectionBackgrounds(main) {
...
+ */
+export function decoratePeople(main) {
+  main.querySelectorAll('.section.people').forEach((section) => {
+    const grid = section.querySelector('div.default-content-wrapper');
+    const peopleFragment = createTag('div', { class: 'people-intro' });
+    const row1Text = createTag('div', { class: 'row1-text' });
+    const row2Text1 = createTag('div', { class: 'row2-text1' });
+    const row2Text2 = createTag('div', { class: 'row2-text2' });
+
+    // Expect:
```
Comment (tmathern) [inline scripts/scripts.js:417]: Do we need to keep the comments?

**Review by tmathern — CHANGES_REQUESTED** [review]
Resizing is off...

File: scripts/scripts.js (hunk 2 above)
Comment (atopper) [reply scripts/scripts.js:417]: We can always get rid of comments, if you don't think it is helpful.  I thought anyone looking at this in the future would be helped by it.
I assumed the JS/CSS was minimized while being published.  No?

File: blocks/vertical-selector/vertical-selector.js [hunk 3]
```diff
@@ -84,12 +84,20 @@ function handleSelectorClick(scope, index) {
     closeButton.classList.remove('vs-active');
   } else {
     const showViewer = viewer.querySelector(`div.fragment-viewer-${index}`);
-    showViewer.classList.add('vs-active');
-    closeButton.classList.add('vs-active');
-    window.dispatchEvent(new Event('drawChart'));
-    const vertSelector = scope.querySelector('.vertical-selector');
-    if (vertSelector) {
-      vertSelector.scrollIntoView(true);
+    if (!showViewer) {
```
Comment (atopper) [inline blocks/vertical-selector/vertical-selector.js:87]: For product 8 (simi), the 3 fragments are not loaded in the vertical selector, but the label is there to click.  This was a quick change to handle the missing fragments nicely - hopefully not needed after all is done but...

**Review by tmathern — APPROVED** [review]

**Pull request merged**